example_dags/example_load_file.py

```diff
@@ -0,0 +1,131 @@
+import pathlib
+from datetime import datetime, timedelta
+
+import sqlalchemy
+from airflow.models import DAG
+
+from astro import sql as aql
+from astro.constants import FileType
+from astro.files import File
+from astro.sql.table import Metadata, Table
+
+CWD = pathlib.Path(__file__).parent
+default_args = {
+    "owner": "airflow",
+    "retries": 1,
+    "retry_delay": 0,
+}
+
+dag = DAG(
+    dag_id="example_load_file",
+    start_date=datetime(2019, 1, 1),
+    max_active_runs=3,
+    schedule_interval=timedelta(minutes=30),
+    default_args=default_args,
+)
+
+
+with dag:
+    # [load_file_example_1_start]
```

Comment:
airflow tag this something like `# [START load_file_example_1]` i.e `start` and `end is at start` of the tag. I think we should follow the same. WDYT?

Reply:
@pankajastro - makes sense I'll start using this in other PRs. Thanks!

Reply:
Done.